pandas/core/frame.py

```diff
@@ -3835,34 +3835,44 @@ def sortlevel(self, level=0, axis=0, ascending=True, inplace=False,
                                inplace=inplace, sort_remaining=sort_remaining)
 
     def nlargest(self, n, columns, keep='first'):
-        """Get the rows of a DataFrame sorted by the `n` largest
-        values of `columns`.
+        """
+        Return the `n` largest rows sorted by `columns`.
+
+        Sort the DataFrame by `columns` in descending order and return the top
```

Comment:
we don't actually sort, can you re-word, we just pick the largest n values

Reply:
Thanks for the review. I was not aware of the internals. Done.